packages/app-utils/src/components/AppBootstrap.tsx

```diff
@@ -19,6 +25,9 @@ export type AppBootstrapProps = {
   /** URL of the server. */
   serverUrl: string;
 
+  /** Version of front-end. */
+  uiVersion: string;
```

Comment:
This prop should be optional - it's not really required, and by adding it we're making this a breaking change for anybody consuming this package (since they'd need to provide `uiVersion` or TypeScript will fail:
```suggestion
  uiVersion?: string;
```

I'd actually go one further, and just take a `logMetadata?: Record<string, unknown>` prop, in case we want to include any other properties in there.